model/shared/src/main/scala/explore/model/ExploreModelValidators.scala

```diff
@@ -43,14 +44,14 @@ object ExploreModelValidators:
   val dithersValidSplitEpi: InputValidSplitEpi[Option[NonEmptyList[DitherNanoMeters]]] =
     InputValidSplitEpi
       .refinedBigDecimal[DitherNanoMetersRange]
-      .toNel()
+      .toNel(".".refined)
       .withErrorMessage("Invalid wavelength dither values".refined)
       .optional
 
   val offsetQNELValidWedge: InputValidWedge[Option[NonEmptyList[Offset.Q]]] =
     MathValidators.truncatedAngleSignedDegrees
       .andThen(Offset.Component.angle[Axis.Q].reverse)
-      .toNel()
+      .toNel(".".refined)
```

Comment:
It seems this might be breaking the ValidWedge Laws.

Reply:
Thanks. it was a "," before, reverting